src/actions/account.js

```diff
@@ -23,9 +26,15 @@ export const handleRedirectUrl = (previousLocation) => (dispatch, getState) => {
     }
 }
 
+const clearTempData = () => {
+    setTempAccount('')
```

Comment:
What is "temp account"? Why do we need special handling for "temp account" vs normal account?

Reply:
We no longer create the account when a user submits a new accountId, so we need to store their desired accountId until after they have completed the recovery setup.